guide/data-management-issues.rst

```diff
@@ -1,107 +1,127 @@
+.. _data-distribution:
 
 Data management in DART	
 =======================
 
 One of the more challenging aspects of an ensemble Data Assimilation (DA) system 
 is the need to manage large amounts of memory to store ensembles of the model state.
 
-Most contemporary large models run in parallel on multi-processor computer systems 
-and distribute the
-data across multiple memory nodes to support finer grids, smaller timesteps and
-longer modeling time periods.  Common computer science strategies include using
-shared memory on individual nodes and using the Message Passing Interface (MPI) 
-libraries on distributed memory nodes.
+Most modern large-scale models run in parallel on multi-processor computer systems, 
+distributing data across multiple memory nodes to support finer spatial grids, 
+smaller time steps, and longer simulation periods. Common strategies include using 
+shared memory within individual nodes and the Message Passing Interface (MPI) 
+on distributed-memory systems.
 
 Ensemble DA exacerbates this memory problem by requiring multiple copies,
 often 20-100x, of the model data to do the assimilation.
 
-DART uses the MPI libraries to distribute ensembles of model state data across
-distributed memory nodes.  For models with small amounts of data the code can be 
-compiled and run as a serial program but when compiled with MPI 
-it can scale up to 10,000s of nodes using Giga to Petabytes of memory.
+DART uses the Message Passing Interface (MPI) to distribute ensembles of 
+state and extended state data across processors and nodes in a distributed-memory system. 
+For models with relatively small memory demands, DART can be compiled and run as a serial program. 
+However, when compiled with MPI, it can scale to tens of thousands of processors 
+and handle memory requirements ranging from gigabytes to petabytes.
 
-Memory usage and internode communication time are mutually incompatible 
-items to minimize.  DART has different strategies that can be selected
-at runtime to use less memory per node at the cost of more time spent in communication
-of data between nodes, or use more memory per node and minimize communication time.
+There are three phases in DART that require access to the whole state vector:
 
-The following descriptions detail the different phases of the main assimilation
-program in DART, called ``filter``, and what options exist for memory layout
-and management.
+1. IO - reading and writing of the model state.
+2. Forward Operator (FO) computation. 
+3. Conversion of of a observation or state location from one vertical coordinate system to another.
```

Comment:
bonus 'of' should be removed.

Reply:
fixed in ca2c67c43f85af9bc744c537aefbe3f65e786e0b